packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -77,11 +77,10 @@ $link-hover-color: darken($link-color, 15%);
 //== Typography
 //## Change your font family, weight, line-height, headings and more (used in components/typography)
 
-// Font Family Import (Used for google font plugin in theme creater https://ux.mendix.com/theme-creator.html)
+// Font Family Import (Used for google font plugin in theme creater)
 $font-family-import: "https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700";
 @if $font-family-import != false {
-    // Only import, if the import is set
-    @import url($font-family-import);
+    @import url("\"#{$font-family-import}\""); // the `url` function removes the outer quotes, escape inner quotes so that compiled value is a string literal.
```

Comment:
Some #magic right here 😄 

The conditional import works fine this way?

Reply:
Yeah 😄 It results in a proper css @import

e.g. `@import "http://google-font.com?family=custom&variant=300"`

Reply:
`#{}` is sass's interpolation syntax, btw

Reply:
Is there an easy way for us to verify/test this output?

Reply:
We can test with another google font. 
"https://fonts.googleapis.com/css?family=Bitter:300,400,600,700"; its a sans-serif font so has noticeable differences from open sans